.pre-commit-config.yaml

```diff
@@ -96,12 +80,3 @@ repos:
       - id: isort
         name: isort except __init__.py
         exclude: /__init__\.py$
-  - repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v2.5.0
-    hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-      - id: check-json
-        files: \.(bowerrc|jshintrc|json)$
-      - id: check-yaml
-      - id: debug-statements
```

Comment:
Why we are removing this part?

Reply:
Added to the above repo with same function